packages/controller/src/lib/letsencrypt.js

```diff
@@ -177,9 +177,9 @@ async function createServerAsync(app, settings, certificates, leSettings, log, a
                             httpServer.listen(settings.lePort, bind, () => {
                                 log.info(`Challenge server listening on port ${settings.lePort}`);
                                 log.info(
-                                    `If something is not working and your adapter is not reachable anymore you can turn off HTTPS with executing "iobroker ${
+                                    `If something is not working and your adapter is not reachable anymore you can turn off HTTPS with executing "iobroker set ${
```

Comment:
hm ... if i look at it that way ... "adapter not reachable"? hhhmm ... "adapter webpage not reachable"? or such

Reply:
yeah just fixed the command in the comment, we can rephrase if you like, makes sense